internal/cortex/querier/queryrange/query_bytes_fetched.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) The Cortex Authors.
+// Licensed under the Apache License 2.0.
+
+package queryrange
+
+import (
+	"strconv"
+)
+
+const QueryBytesFetchedHeaderName = "M3-Fetched-Bytes-Estimate"
```

Comment:
This name is compatible with M3 and rule manager code, can you add a comment here in case people might be confused?